packages/flutter_driver/lib/src/extension.dart

```diff
@@ -123,7 +123,7 @@ class FlutterDriverExtension {
     return _onFrameReadyStream;
   }
 
-  Future<Health> getHealth(GetHealth command) async => new Health(HealthStatus.ok);
+  Future<Health> getHealth(Command command) async => new Health(HealthStatus.ok);
```

Comment:
Good catch. Fixed. In fact, fixed all of them. All methods corresponding to commands are routed to internally.